Products/Zuul/infos/component/__init__.py

```diff
@@ -28,6 +28,10 @@ def device(self):
         return self._object.device()
 
     @property
+    def deviceName(self):
+        return self._object.device().getId()
```

Comment:
If you legitimately want this to be the name, you should do `self._object.device().titleOrId()`